contributors/guide/style-guide.md

```diff
@@ -0,0 +1,653 @@
+    there is a large content shift or "chapter" change.
+* Follow the Header hierarchy of `H2` > `H3` > `H4` > `H5` > `H6`.
+* Use sentence-style capitalization in titles (first word and proper nouns).
+* Avoid using special characters.
+* Leave exactly 1 new line after a heading.
+* Avoid using links in headings.
+
+**[Horizontal rules:](#horizontal-lines)**
+* Use three dashes (`---`) to denote a horizontal rule.
+* Use a horizontal rule (`---`) to logically separate large sections.
+
+**[Line length:](#line-length)**
+* Prefer an 80 character line limit.
+
+**[Links:](#links)**
+* Prefer using reference style links over inline style links.
+* When linking within the same directory, use a relative link.
+* When linking to a document outside of the current directory, use the absolute
+  path from the root of the repository.
+* When linking to a file in another Kubernetes github repository, use the
+  `k8s.io` url shortener.
+  * git.k8s.io -> github.com/kubernetes
+  * sigs.k8s.io -> github.com/kubernetes-sigs
+
+**[Lists:](#lists)**
+* Capitalize the first character of each entry unless the item is explicitly
+  case sensitive.
+* End each entry with a period if it is a sentence or phrase.
+* Use a colon (`:`) to separate a list item name from the explanatory text.
+* Leave a blank line after each list.
+* Use `*` for unordered lists.
+* For ordered lists repeating `1.` may be used.
+* When inserting a code block into an ordered list, indent (space) an additional
+  two times.
+
+**[Metadata:](metadata)**
+* Include a yaml metadata header at the beginning of the document.
+* Metadata must include the `title` attribute.
+
+**[Tables:](#tables)**
+* Use tables for structured information.
+* Tables do not need to adhere to the suggested line length.
+* Avoid long inline links.
+* Do not use excessively wide tables.
+
+---
+
+## Content design, formatting, and language
+
+### Contact information
+
+* Use official Kubernetes contact information.
+  * Use official community contact email addresses. There should be no personal
+    or work contact information included in public documentation; instead use
+    addresses like the [SIG Google groups] or managed accounts such as
+    community@kubernetes.io. 
+  * **Good example:** community@kubernetes.io
+  * **Bad example:** bob@example.com
+
+
+### Dates and times
+
+The Kubernetes Contributor Community spans many regions and timezones. Following
+a consistent pattern and avoiding shorthand improves the readability for every
+member.
+
+* Format dates as `month day, year`. (December 13, 2018)
+  * **Good example:** October 24, 2018
+  * **Bad example:** 10/24/18
+* When conveying a date in numerical form, use [ISO 8601] Format: `yyyy-mm-dd`.
+  * **Good example:** 2018-10-24
+  * **Bad example:** 10/24/18
+* Use the 24 hour clock when referencing time.
+  * **Good example:** 15:30
+  * **Bad example:** 3:30pm
+* Times should be expressed in the UTC timezone in addition to any local times.
```

Comment:
Had a bit of a bikeshed in meeting, just noting it so it doesn't get lost, should figure out recommendations for recurring events re: timezones.

Reply:
It could be updated to be something like "Time should be expressed in a daylight savings  agnostic time such as ET or PT." and require linkout to the timezone converter service.

Reply:
Roughly restating what I heard Noah say during the meeting:
- single point in time events should be expressed in an absolute timezone (e.g. use UTC instead of GMT, use EDT instead of ET)
- recurring events should be expressed in the most appropriate local timezone that follows DST changes (e.g. use GMT instead of UTC, or use PT instead of PST/PDT)

Reply:
I like that idea. Should cover all our use-cases.